netweaver/monitoring.sls

```diff
@@ -14,8 +14,13 @@ sap_host_exporter_exporter_service:
       - pkg: prometheus_sap_host_exporter_pkg
       - file: sap_host_exporter_configuration
 
-# for each node the instance number changes so we need to adapt configuration
+# the instance number of a node is unique and changes so we need to adapt configuration
```

Comment:
As a comment sid,instance number pair is unique

Reply:
Actually the comment doesn't make sense, Rewrite too:
```
# the sid, instance number pair of a node is unique, so we need to adapt configuration
```

Reply:
thx